### PR TITLE
Added Bookwatch Plugin 

### PR DESCRIPTION
**Pull request by AlexTs10**
Added the bookwatch plugin in the community-plugins.json

### CODE REVIEW

File: community-plugins.json [hunk 1]
```diff
@@ -605,5 +605,23 @@
...
     "chat_prompt": "You are Nik Shevchenko, 24yo founder of Based Hardware. Nik built Friend AI necklace that transcribes your conversations and gives you summaries and personalized advice. Nik also created openglass that takes pictures every few seconds and gives you personalized AI. Nik is a straightforward Russian guy. Nik responds in a short format, getting directly to the point. As a first message, Nik likes to write diss tracks. At the beginning of each conversation, respond with a diss track about the user",
     "deleted": false
-  }
+  },
+  {
+    "id": "bookwatch-friend-plugin-idx",
+    "name": "BookWatch Friend Plugin",
+    "author": "Alex Toska - Miran Antamian",
+    "description": "This plugin enables you to add bookmarks for books you want to watch in BookWatch later",
+    "image": "https://cdn.prod.website-files.com/6696811effea0ba47d64eca9/66984b3ae54fc5b0fc78a34f_Bookwatch%20Logo%20Trademark%20(5).png",
```
Comment (josancamon19) [inline community-plugins.json]: This looks amazing!

Thank you @AlexTs10 for your first contribution!

A slight change, the image field, should be a file referenced in https://github.com/BasedHardware/Omi/tree/main/plugins/logos

Also, can you please submit a video showing off this? Thanks!

**Review by josancamon19 — CHANGES_REQUESTED** [review]

**Comment by AlexTs10** [comment]
Changed the image field, will also try to record a video.

**Comment by AlexTs10** [comment]
Is the video required in order to get accepted ?

**Comment by josancamon19** [comment]
Hi unfortunately yes, a video is reuqired to see how the plugin works, thanks @AlexTs10

**Comment by josancamon19** [comment]
or at least a video from postman requests

**Comment by AlexTs10** [comment]
Hey @josancamon19  added the video in the setup_instructions_file_path
Also you can check it here: https://drive.google.com/file/d/1zJqOfghhuBeLzu7XySpEZEIklnYndpdA/view?usp=sharing

Happy to answer any questions if needed.

**Comment by josancamon19** [comment]
LGTM

**Review by josancamon19 — APPROVED** [review]

**Review by josancamon19 — APPROVED** [review]

**Comment by josancamon19** [comment]
I would change the image, as it will not look great on the app.

**Comment by josancamon19** [comment]
CleanShot 2024-08-29 at 01 20 33@2x
Got this btw

**Comment by AlexTs10** [comment]
Will check, probably its because you didn't do the setup.

**Comment by AlexTs10** [comment]
Hey, i wanted to ask why i can't see the plugin in the app since it passed the pr ?

**Pull request merged**